### PR TITLE
Update dev guides to Flux v2.3

### PR DESCRIPTION
**Pull request by stefanprodan**
Update API versions in dev guides

### CODE REVIEW

File: content/en/flux/gitops-toolkit/packages.md [hunk 1]
```diff
@@ -187,7 +188,7 @@ func main() {
   ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
   defer cancel()
 
-  // create a Helm repository pointing to Bitnami
+  // create a Helm repository pointing to https://kubernetes.github.io/ingress-nginx
```
Comment (souleb) [inline content/en/flux/gitops-toolkit/packages.md:191]: why the change here? The repository points to bitnami still.

Comment (stefanprodan) [reply content/en/flux/gitops-toolkit/packages.md]: Good catch, i wanted to change the repo but installing ingress-nginx in the default ns doesn't make sense. We should rewrite this with OCIRepo, but that's for another time.

**Review by souleb — APPROVED** [review]
LGTM!
